src/main/java/com/jjoe64/graphview/DefaultLabelFormatter.java

```diff
@@ -62,6 +62,24 @@ public DefaultLabelFormatter(NumberFormat xFormat, NumberFormat yFormat) {
         mNumberFormatter[1] = xFormat;
     }
 
+    /**
+     * get a number format for x axis
+     *
+     * @return an object of NumberFormat type for x axis
+     */
+    public NumberFormat getXFormat() {
+        return mNumberFormatter[1];
```

Comment:
Why is the x format stored at index 1 and not at index 0? Not a big deal, but it is usually a convention to store x and y in that order.

Reply:
@mbalasciaps That is a good question! :-)
Here is a part of the source code of the GraphView project:
com/jjoe64/graphview/DefaultLabelFormatter.java
```    public DefaultLabelFormatter(NumberFormat xFormat, NumberFormat yFormat) {
        mNumberFormatter[0] = yFormat;
        mNumberFormatter[1] = xFormat;
    }```
I’m also wondering why the author implemented it in such way.